tests/cpp_api_tests/rocAL_audio_unittests/rocAL_audio_unittests.cpp

```diff
@@ -109,16 +109,16 @@ bool verify_output(float *dst_ptr, long int frames, std::string case_name, int m
 
     if (fin.fail()) {
         std::cout << "Error: Failed to read from the input binary file\n";
-        return 1;
+        return 0;
     }
 
     fin.close();
 
     int matched_indices = 0;
-    for (int i = 0; i < max_samples; i++) {
-        for (int j = 0; j < frames; j++) {
+    for (int i = 0; i < frames; i++) {
```

Comment:
All these changes should be done in Spectrogram PR itself, Please sync with Swetha and add these in Spec PR